src/main/java/com/codeborne/selenide/CollectionCondition.java

```diff
@@ -103,12 +96,42 @@ public static CollectionCondition exactTexts(List<String> expectedTexts) {
     return new ExactTexts(expectedTexts);
   }
 
+  private static class ExplainedCollectionCondition extends CollectionCondition {
```

Comment:
Also new class and method lack of documentation.

Reply:
Added overall explanation